pkg/rpc/server_test.go

```diff
@@ -113,6 +128,7 @@ func TestHandler(t *testing.T) {
 				t.Errorf("could not read response from the request: %s", tc.rpcCall)
 			}
 
+			fmt.Println(string(bytes.TrimSpace(body)))
```

Comment:
t.Log?

Reply:
I used the `Println` for testing purpose but forgot to remove it later. I was planning to remove it in the next pull request. But will do now.  :sweat_smile:

Reply:
I think you could use t.Log /t.Logf for debugging.. this is the right approach in golang.